salt/modules/dockermod.py

```diff
@@ -6674,6 +6688,27 @@ def call(name, function, *args, **kwargs):
         run_all(name, subprocess.list2cmdline(rm_thin_argv))
 
 
+def apply_(name, mods=None, **kwargs):
+    '''
+    .. versionadded:: Flourine
+
+    Apply states! This function will call highstate or state.sls based on the
+    arguments passed in, state.apply is intended to be the main gateway for
+    all state executions.
```

Comment:
Should the wording here be modified to reference the `docker` module instead of `state`?

Reply:
I am just going to remove the word `state` because this will actually be referenced as `state.apply` when called through the docker proxy minion.

Reply:
OK, cool :+1: